eth/api_backend.go

```diff
@@ -290,6 +291,39 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 	return b.eth.txPool.Add([]*txpool.Transaction{{Tx: signedTx}}, true, false)[0]
 }
 
+func (b *EthAPIBackend) SendBundle(ctx context.Context, bundle *types.Bundle) error {
+	return b.eth.txPool.AddBundle(bundle)
+}
+
+func (b *EthAPIBackend) MinimalBundleGasPrice() *big.Int {
+	gasFloor := big.NewInt(b.eth.config.Miner.MevGasPriceFloor)
+	gasPrice := b.eth.TxPool().MinimalBundleGasPrice()
+	if gasPrice.Cmp(gasFloor) < 0 {
+		return gasFloor
+	}
+	return gasPrice
+}
+
+func (b *EthAPIBackend) BundlePrice() *big.Int {
+	bundles := b.eth.txPool.AllBundles()
+
+	sort.SliceStable(bundles, func(i, j int) bool {
+		return bundles[j].Price.Cmp(bundles[i].Price) < 0
+	})
+
+	gasFloor := big.NewInt(b.eth.config.Miner.MevGasPriceFloor)
+	idx := len(bundles) / 2
+	if bundles[idx].Price.Cmp(gasFloor) < 0 {
+		return gasFloor
+	}
+
+	return bundles[idx].Price
+}
+
+func (b *EthAPIBackend) UnregisterMevValidator(validator common.Address) {
```

Comment:
Let us keep it simple, just config validator in config file , dont change it dynamically in day1

Reply:
This function is used to remove validator when got reported issue that the target validator is not in mev mode